py/server/deephaven/time.py

```diff
@@ -588,9 +591,11 @@ def to_time(dt: Union[None, LocalTime, ZonedDateTime]) -> Optional[datetime.time
         if dt is None:
             return None
         elif isinstance(dt, LocalTime.j_type):
-            return datetime.time(dt.getHour(), dt.getMinute(), dt.getSecond(), dt.getNano() // _nanos_per_micro)
+            hour, minute, second, nano = _JPythonTimeComponents.getComponents(dt)
+            return datetime.time(hour, minute, second, nano // _nanos_per_micro)
```

Comment:
Didn't catch this when reviewing #4388, but we probably should rename `_nanos_per_micro` to `_NANOS_PER_MICRO` to be more PEP8 compliant.